lib/alaveteli_mail_poller.rb

```diff
@@ -35,9 +35,11 @@ def poll_for_incoming
   def self.poll_for_incoming_loop
     if AlaveteliConfiguration.production_mailer_retriever_method == 'pop'
       poller = new
+      Rails.logger.info "Starting #{ poller } polling loop"
```

Comment:
These do log to `log/development.log`.

I guess on reflection that spreading the log messages over the app log and the poller log isn't ideal, but by using `puts` you have no control over the log level. You can set up a new logger for the class that logs to `stdout`, which might be worth doing:

```ruby
  logger = ActiveSupport::Logger.new(STDOUT)
  logger.formatter = Rails.application.config.log_formatter
  logger = ActiveSupport::TaggedLogging.new(logger)
```

Anyway, this was just a suggested improvement as it was difficult to see what was happening in a development context. I'll make PRs for suggested commits in future.

Feel free to drop this and/or a47258099c30a76f6c12b2f7ab8f1520565d8558.